seive.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+function seiveOriginal(n) {
+  let primes = new Array(n - 1).fill(true);
+  [primes[0], primes[1]] = [false, false];
+  for (let i = 2; i <= n; i++) {
+    for (let j = i * i; j <= n; j += i) {
+      primes[j] = false;
+    }
+  }
+
+  primes.forEach((el, i) => {
+    if (el) console.log(i);
+  });
+}
+
+function blocksSeive(n) {
+  let nPrime = new Array(n).fill(false);
```

Comment:
The algorithm's point is to not use the array of length `n` so try to make it work without it.